specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/examples/Updates/Get.json

```diff
@@ -15,15 +15,28 @@
         "location": "redmond",
         "tags": {},
         "properties": {
-          "installedDate": "2018-03-03T08:09:12.8824552Z",
+          "displayName": "displayName",
           "description": "MAS Update 1.0.180302.1",
+          "version": "1.0.180302.1",
+          "availabilityType": "online",
           "state": "Installed",
-          "kbLink": "https://aka.ms/azurestackupdate",
-          "minVersionRequired": "1.0.180103.2",
+          "releaseLink": "https://aka.ms/azurestackupdate",
+          "prerequisites": [],
+          "packageSizeInMb": 23345,
           "packagePath": "\\\\SU1FileServer\\SU1_Infrastructure_2\\Updates\\Packages\\Microsoft1.0.180302.1",
-          "packageSizeInMb": 2144,
-          "version": "1.0.180302.1",
-          "publisher": "Microsoft"
+          "publisher": "Microsoft",
+          "updateStateProperties": {
+            "downloadPercentage": 100,
+            "installedDate": "2018-03-03T08:09:12.8824552Z"
+          },
+          "azsUpdateProperties": {
+            "minVersionRequired": "1.0.180103.2",
+            "minOemVersionRequired": "2.0.1.0"
```

Comment:
Should use realistic versions - these aren't even formatted properly in terms of where the YYMM goes.  Apply this across all samples to increase their relevance
```
"minVersionRequired": "1.2008.0.43",
"minOemVersionRequired": "2.1.1908.0"
```